nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/admin/admin-ai-export.html

```diff
@@ -0,0 +1,169 @@
+            <paper-input id="outputs" label="[[i18n('admin.ai.outputs')]]" value="{{outProps}}"></paper-input>
+            <paper-input id="split" label="[[i18n('admin.ai.split')]]" value="{{splitProp}}"></paper-input>
+            <paper-button on-tap="_recompute">[[i18n('admin.ai.stats.action')]]</paper-button>
+            <paper-button on-tap="_export">[[i18n('admin.ai.export.action')]]</paper-button>
+        </nuxeo-card>
+        <nuxeo-card heading="[[i18n('admin.ai.stats')]]">
+            <div class="row-container">
+                <span class="totalLabel">[[_computeLabel(_stats, 'total', i18n)]]</span>
+                <span class="totalLabel">[[_computeLabel(_stats, 'count', i18n)]]</span>
+            </div>
+            <nuxeo-data-table id="statsResult" items="[[_filter(_stats)]]">
+                <nuxeo-data-table-column name="[[i18n('admin.ai.stats.field')]]" flex="10">
+                    <template>
+                        [[item.field]]
+                    </template>
+                </nuxeo-data-table-column>
+                <nuxeo-data-table-column name="[[i18n('admin.ai.stats.type')]]" flex="10">
+                    <template>
+                        [[item.type]]
+                    </template>
+                </nuxeo-data-table-column>
+                <nuxeo-data-table-column name="[[i18n('admin.ai.stats.value')]]" flex="50">
+                    <template>
+                        [[_displayVal(item)]]
+                    </template>
+                </nuxeo-data-table-column>
+            </nuxeo-data-table>
+        </nuxeo-card>
+        <nuxeo-card heading="[[i18n('admin.ai.export')]]">
+            <div class="row-container">
+                [[_exportText]]
+            </div>
+        </nuxeo-card>
+    </template>
+    <script>
+        Polymer({
+            is: 'nuxeo-admin-ai-export',
+            behaviors: [Nuxeo.LayoutBehavior],
+            properties: {
+                inProps: {
+                    type: String,
+                    value: "dc:title,file:content"
+                },
+                outProps: {
+                    type: String,
+                    value: "dc:lastContributor"
+                },
+                splitProp: {
+                    type: String,
+                    value: "80"
+                },
+                query: {
+                    type: String,
+                    value: "SELECT * FROM Document WHERE ecm:primaryType = 'File'"
+                },
+                _exportText: {
+                    type: String,
+                    value: ""
+                },
+                _stats: {
+                    type: String
+                }
+            },
+            _displayVal: function (val) {
+                if (val.numericValue != null) {
+                    return val.numericValue;
+                }
+                if (val.type === 'terms') {
+                    var toReturn = '';
```

Comment:
And single quotes here. We usually stick with only way of defining strings and by convention we use single quotes.